stable/dex/values.yaml

```diff
@@ -159,12 +208,17 @@ config:
 #    secret: ZXhhbXBsZS1hcHAtc2VjcmV0
 #
   enablePasswordDB: true
-# staticPasswords:
-#  - email: "admin@example.com"
-#    # bcrypt hash of the string "password"
-#    hash: "$2a$10$2b2cU8CPhOTaGrs1HRQuAueS7JTT5ZHsHSzYiFPm1leZck7Mc8T4W"
-#    username: "admin"
-#    userID: "08a8684b-db88-4b73-90a9-3cd1661f5466"
+#  staticPasswords:
+#   - email: "admin@example.com"
+#     # bcrypt hash of the string "password"
+#     hash: "$2a$10$2b2cU8CPhOTaGrs1HRQuAueS7JTT5ZHsHSzYiFPm1leZck7Mc8T4W"
+#     username: "admin"
+#     userID: "08a8684b-db88-4b73-90a9-3cd1661f5466"
+
+  frontend:
```

Comment:
should we move this setting to `kubernetes-base-addons`?  I like the pattern where we keep the chart close to upstream and have d2iq specific settings in `kubernetes-base-addons`

Reply:
@GoelDeepak i was debating that initially myself. I decided to bake in the chart because this is a fork anyway (i.e., D2iQ flavor of dex) with some other changes (e.g., certificate, etc.).